examples/rig-kafka-pub-sub-example/docker-starter/docker-compose.yml

```diff
@@ -0,0 +1,48 @@
+version: "3"
+services:
+  zk:
+    image: confluentinc/cp-zookeeper:5.0.0
+    container_name: zk
+    ports:
+      - 2181:2181
+    environment:
+      ZOOKEEPER_CLIENT_PORT: 2181
+      ZOOKEEPER_TICK_TIME: 2000
+    restart: on-failure:5
+
+  kafka:
+    image: confluentinc/cp-kafka:5.0.0
+    container_name: kafka
+    ports:
+      - 9092:9092 # container communication
+      - 9094:9094 # localhost communication
+    environment:
+      KAFKA_ZOOKEEPER_CONNECT: zk:2181
+      KAFKA_ADVERTISED_LISTENERS: INSIDE://kafka:9092,OUTSIDE://localhost:9094
+      KAFKA_LISTENERS: INSIDE://0.0.0.0:9092,OUTSIDE://0.0.0.0:9094
+      KAFKA_LISTENER_SECURITY_PROTOCOL_MAP: INSIDE:PLAINTEXT,OUTSIDE:PLAINTEXT
+      KAFKA_INTER_BROKER_LISTENER_NAME: INSIDE
+      KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR: 1
+      KAFKA_GROUP_INITIAL_REBALANCE_DELAY_MS: 0
+      KAFKA_AUTO_CREATE_TOPICS_ENABLE: "true"
+      KAFKA_NUM_PARTITIONS: 4
+    restart: on-failure:5
+
+  reactive-interaction-gateway:
+    container_name: reactive-interaction-gateway
+    image: accenture/reactive-interaction-gateway:2.1.1
+    environment:
+      - LOG_LEVEL=debug
+      - KAFKA_SOURCE_TOPICS=rig,rig-consumed
+      - PROXY_KAFKA_REQUEST_TOPIC=rig
+      # - PROXY_KAFKA_RESPONSE_TOPICS=rig
+      - KAFKA_BROKERS=kafka:9092
+      - API_HTTP_PORT=7010
+      - INBOUND_PORT=7000
+      - PROXY_CONFIG_FILE=proxy/rig-proxy.json
+    ports:
+      - 7000:7000
+      - 7010:7010
+    volumes:
+      - ./rig-proxy.json:/opt/sites/rig/lib/rig_inbound_gateway-2.1.1/priv/proxy/rig-proxy.json
```

Comment:
Current version can do better lookup, so this should be enough `- ./rig-proxy.json:/rig-proxy.json`.